src/ApiProblem.php

```diff
@@ -431,12 +431,24 @@ public function asXml(bool $pretty = false)
      *
      * @return array
      *   The API problem represented as an array.
-    */
+     */
     public function asArray() : array
     {
         return $this->compile();
     }
 
+    /**
+     * Supports rendering this problem as a JSON using the
+     * json_encode() function.
```

Comment:
The description line is short enough that it should be a single line.  No wordwrap.